Server/Scripts/configure.ps1

```diff
@@ -11,6 +11,7 @@ $env:OCTOPUS_INSTANCENAME=$OctopusInstanceName
   $masterKeySupplied = ($masterKey -ne $null) -and ($masterKey -ne "")
   $octopusAdminUsername = $env:OctopusAdminUsername
   $octopusAdminPassword = $env:OctopusAdminPassword
+  $credentialsSupplied=($octopusAdminPassword -ne $null -or $octopusAdminUserName -ne $null)
```

Comment:
another nulls on the left thing. Can be addressed later if we put Pester testing on this repo